samplomatic/builders/template_state.py

```diff
@@ -85,22 +93,22 @@ def construct_for_circuit(cls, circuit: QuantumCircuit) -> Self:
 
         return cls(template_circuit, qubit_map, param_iter, [])
 
-    def append_remapped_gate(
-        self, instr: CircuitInstruction, target_circuit: QuantumCircuit | None = None
-    ) -> ParamSpec:
-        """Remap the parameters and qubits of an instruction gate and append it to the circuit."""
-        if target_circuit is None:
-            target_circuit = self.template
+    def qubits(self, idxs: Iterable[int] | None = None) -> Sequence[Qubit]:
+        """Return the qubits in the template at the given indices."""
```

Comment:
This method should probably get a test.

```suggestion
        """Return the qubits in the template at the given indices.
        
        Args:
            idxs: The indices of qubits to collect, or ``None`` to get all of them.
            
        Returns:
            A sequence of qubits, in order.
        """
```